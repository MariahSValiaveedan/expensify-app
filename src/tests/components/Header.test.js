import React from 'react';
import {shallow} from 'enzyme';
import Header from '../../components/Header';
//import toJSON from'enzyme-to-json';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';



Enzyme.configure({
    adapter : new Adapter()
});

test('should render header', () => {
   const wrapper = shallow(<Header />);
   expect((wrapper)).toMatchSnapshot();
   
   
    /* const renderer=new ReactShallowRenderer();
    renderer.render(<Header />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();*/
    
});
