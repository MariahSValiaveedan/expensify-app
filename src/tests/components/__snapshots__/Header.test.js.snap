// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
     Go to Dashboard 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
      add expense  
  </NavLink>
</header>
`;
